IMC_WebDev/MiniCityBlog.md

```diff
@@ -0,0 +1,96 @@
+
+After we create all of our objects and their “children”, it is time to close our body and our html tag by using /body and /html.
+
+When it’s time to design our objects in CSS, we MUST use the exact name we created for them in our HTML. For example, if I want to change the color of the bridge in my city, I need to name it “bridge” in my CSS, just like how it is in my HTML.
+
+Next, let’s prepare our editor for the CSS portion of the code. We do this by establishing a style tag. Since we will be using CSS, our style tag will look like this: <img src=https://lh5.googleusercontent.com/kNUcMVEmTr0O0GyTUlYwlJro5lYXiehFh5gwS_B4xZujFUkrVZEn9pIjoA-venhZlkqoWjYBlksmDZ4o0-SEaem_FFR072jiqH26Svuy4ga0ACzWjt4m-5JQG1FI_mBGbIDppin0>
+
+We will be writing all of our CSS in our head portion of our code.
+
+To style our webpage, we will need to learn the CSS syntax. A CSS rule-set consists of a selector and a declaration block.
+<img src=https://lh3.googleusercontent.com/3UcaVdjNU8MuMn4Feims4asq1epCgIOEwJX2goOH8SZE2eP2IKz5vRvCuSL9howvCc_Co_G2TE8Tqe5pIPgTAITV9KB28hqPEWXLzvZZ7xTnF1GNBohcY75WK00KBHIfbJ4nufVR>
+
+Apart from color and font-size, these are some of the properties you will require to get started:
+
+Position: It specifies the type of positioning method used for an element: static, relative, fixed, absolute or sticky. 
+Display: it handles whether an element is shown or not. If you choose display: none, the element will not be visible.
+Background-color: It determines the background color of the element. A color can be specified by a valid color name - like “black”, a hexadecimal value - like “ffffff” or an rgb value - like “rgb(255,0,0)”.
+Height and Width: These are properties used to set the height and width of the element.
+Transform: This property is used to make 2D transformations. The most commonly used transformations are: rotate() and skew.
+rotate(): It rotates an element clockwise or counter-clockwise according to a given degree. For example, 
+transform: rotate(20deg);
+ skew(): skews an element along the X and Y-axis by the given angles. For example, this example skews the element 30 degrees along the X-axis, and 50 degrees along the Y-axis.
+transform: skew(30deg, 50deg);
+Top: It affects the vertical position of the element. The above example sets the element 18px below the top edge of its nearest positioned ancestor.
+Left:  It affects the horizontal position of the element. The above example sets the element 90px from the left edge of its nearest positioned ancestor.
+Z-Index: It specifies the stack order of an element such that an element with a greater value is drawn before an element with a lower value. For example, an element with a z-index of -990 will be drawn before an element which has a z-index of -1000.
```

Comment:
Z-index isn't that straightforward of a concept. Really dive into this and carefully explain what "stacking contexts" are for example. If you look up z-index on YouTube, the tutorials are anywhere from about 10 to 20 minutes on this topic alone. You definitely shouldn't spend an entire paragraph on z-index alone, but you can't properly explain it in one line either. At the very least, try to paraphrase the most important points in a 2-3 sentences.